pymc/distributions/multivariate.py

```diff
@@ -1855,22 +1855,21 @@ class KroneckerNormal(Continuous):
 
     Parameters
     ----------
-    mu: array
+    mu : tensor_like of float
         Vector of means, just as in `MvNormal`.
-    covs: list of arrays
+    covs : list of arrays
         The set of covariance matrices :math:`[K_1, K_2, ...]` to be
         Kroneckered in the order provided :math:`\bigotimes K_i`.
-    chols: list of arrays
+    chols : list of arrays
         The set of lower cholesky matrices :math:`[L_1, L_2, ...]` such that
         :math:`K_i = L_i L_i'`.
-    evds: list of tuples
+    evds : list of tuples
```

Comment:
@Sayam753 @purna135 Is `KroneckerNormal` one of the distributions being updated as part of your GSoC work? Has it been or will it be?

Reply:
Hi @OriolAbril 
Yes. The plan is to work on multivariate distributions in PyMC post mid-evaluations.

Reply:
Thanks, @vitaliset  it is ok then to leave the types as is and count on Sayam and Purna to update them as part of the planned improvements.